Dockerfile

```
@@ -0,0 +1,44 @@
+# Thsi dockerfile builds a docker image for the example site.
+
+# Use an official Python runtime as a parent image
+FROM python:3.8
+LABEL maintainer="hello@wagtail.io"
+
+# Set environment varibles
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_ENV dev
+
+# we're only going to copy the requirements for pip first, so if we're working
+# on the example site and testing docker builds we don't have to rebuid the
+# "pip requirements layers" every time we change a file.
+
+# copy local pip dependences
+COPY ./grapple /code/grapple
+COPY ./setup.cfg /code/setup.cfg
+COPY ./setup.py ./code/setup.py
+COPY ./requirements-channels.txt /code/requirements-channels.txt
+COPY ./requirements.txt /code/requirements.txt
```

Suggestion: this step could be written like this
```suggestion
COPY requirements.txt setup.cf setup.py requirements-channels.txt requirements.txt /code/
```